doc/chips/9.rst

```diff
@@ -324,11 +324,10 @@ This will be a fully automated step in which mason does the following:
 If the user has manually edited their manifest file, they may update the
 lock file using ``mason update`` from within the project directory. Otherwise,
 the lock file is updated automatically when using ``mason add`` or ``mason rm``.
-(see: Syncing Commands)
+(see Syncing Commands)
```

Comment:
Maybe double check this, but I think ``(see `Syncing Commands`_ )`` will add a cross-reference.